lib/mustache-parser.pegjs

```diff
@@ -21,6 +22,14 @@ newMustache = name:newMustacheStart m_* attrs:(m_bracketedAttrs / newMustacheAtt
   return ret;
 }
 
+m_blockParams = m_blockStart m_* params:blockParamName+ "|" {
+  return params;
+}
+
+m_blockStart = "as" m_* "|"
```

Comment:
@Bestra actually I forgot that PEG doesn't have tokens so my comment was useless :P